src/Jackett.Common/Services/IndexerManagerService.cs

```diff
@@ -231,8 +229,8 @@ public IIndexer GetIndexer(string name)
             if (renamedIndexers.ContainsKey(name))
             {
                 realName = renamedIndexers[name];
-                logger.Warn($"Indexer {name} has been renamed to {realName}. Please, update the URL of the feeds. " +
-                            "This may stop working in the future.");
+                logger.Warn($@"Indexer {name} has been renamed to {realName}. Please, update the URL of the feeds.
+ This may stop working in the future.");
```

Comment:
Sorry if this is a stupid question, but was this supposed to be one line?

Reply:
Well, this line is too long to be readable in the IDE. Will be printed in one line, but the code looks weird. There is not a good solution in C# => https://stackoverflow.com/a/1598507